spec/exasol/rls/RlsMetadataReader_spec.lua

```diff
@@ -0,0 +1,188 @@
+                {
+                    table = "T3",
+                    columns = {
+                        {COLUMN_NAME = "C3_1", COLUMN_TYPE = "BOOLEAN"},
+                        {COLUMN_NAME = "EXA_ROW_TENANT"},
+                        {COLUMN_NAME = "EXA_ROW_ROLES"}
+                    }
+                },
+                {
+                    table = "T4",
+                    columns = {
+                        {COLUMN_NAME = "C4_1", COLUMN_TYPE = "DATE"},
+                        {COLUMN_NAME = "EXA_ROW_GROUP"}
+                    }
+                }
+        )
+        assert.are.same(
+                {
+                    tables = {
+                        {
+                            name = "T3",
+                            columns = {{name = "C3_1", dataType = {type = "BOOLEAN"}}}
+                        },
+                        {
+                            name = "T4",
+                            columns = {{name = "C4_1", dataType = {type = "DATE"}}}
+                        }
+                    },
+                    adapterNotes = "T3:tr-,T4:--g"
+                },
+                reader:read("S")
+        )
+    end)
+
+    local function mock_table_with_single_column_of_type(type)
+        mock_tables("S",
+                {
+                    table = "T",
+                    columns = {{COLUMN_NAME = "C1", COLUMN_TYPE = type}}
+                }
+        )
+    end
+
+    local function assert_column_type_translation(translation)
+        assert.are.same({tables = {{name = "T", columns = {{name = "C1", dataType = translation}}}},
+                         adapterNotes = "T:---"}, reader:read("S"))
+    end
+
+    describe("translates column type:", function()
+        local parameters = {
+            {"BOOLEAN", {type = "BOOLEAN"}},
+            {"DATE", {type = "DATE"}},
+            {"DECIMAL(13,8)", {type = "DECIMAL", precision = 13, scale = 8}},
+            {"DOUBLE PRECISION", {type = "DOUBLE PRECISION"}},
+            {"CHAR(130) UTF8", {type = "CHAR", characterSet = "UTF8", size = 130}},
+            {"CHAR(2000000) ASCII", {type = "CHAR", characterSet = "ASCII", size = 2000000}},
+            {"VARCHAR(70) UTF8", {type = "VARCHAR", characterSet = "UTF8", size = 70}},
+            {"VARCHAR(2000000) ASCII", {type = "VARCHAR", characterSet = "ASCII", size = 2000000}},
+            {"HASHTYPE(5 BYTE)", {type = "HASHTYPE", bytesize = 5}},
+            {"TIMESTAMP", {type = "TIMESTAMP"}},
+            {"GEOMETRY(4)", {type = "GEOMETRY", srid = 4}},
+            {"INTERVAL YEAR(6) TO MONTH", {type = "INTERVAL", fromTo = "YEAR TO MONTH", precision = 6}},
+            {"INTERVAL DAY(9) TO SECOND(5)", {type = "INTERVAL", fromTo = "DAY TO SECONDS",
+                                              precision = 9, fraction = 5}},
+            {"TIMESTAMP WITH LOCAL TIME ZONE", {type = "TIMESTAMP", withLocalTimeZone = true}},
+            {"GEOMETRY", {type = "GEOMETRY", srid = 0}}
```

Comment:
Does it make sense to add tests with default values, e.g. `VARCHAR`, `GEOMETRY`, ...?